### PR TITLE
propValue.trim() sometimes causes TypeError

### PR DESCRIPTION
**Pull request by gfx**
I have errors from HTML2React when it processes user-generated contents.

I guess that's because it can't handle broken styles, and this PR fixes it.

### CODE REVIEW

File: test/get-props-from-attributes.test.js [hunk 1]
```diff
@@ -44,7 +44,19 @@ it("gets props from <span style='...'/> element", () => {
...
             style: {
                 color: 'red',
-                fontWeight: 'bold'
+                fontWeight: 'bold',
+            },
+        });
+});
+
+it("works with invalid styles", () => {
+    const element = toElement('<span style="color:;font-size">foo</span>');
```
Comment (gfx) [inline test/get-props-from-attributes.test.js]: The style is completely broken but even if so HTML2React should not die.

File: src/get-props-from-attributes.js [hunk 2]
```diff
@@ -30,12 +30,11 @@ function getStylePropValue (attrValue) {
   })
 
   return props.reduce((props, prop) => {
-    let [propName, propValue] = prop.split(/:(.+)?/)
+    let [propName, propValue] = prop.split(/:/)
```
Comment (Deschtex) [inline src/get-props-from-attributes.js]: This needs to be `prop.split(/:(.+)?/)` in order to support values that contain colons, e.g. `background-image: url(http://foo.bar)`. I think it makes sense for the result value to be `undefined` for both `color` and `fontSize` in the case of `style="color:;font-size"`.

Comment (gfx) [reply src/get-props-from-attributes.js]: Right!

Fixed with a test case: 431e562

**Review by Deschtex — APPROVED** [review]
👍

**Pull request merged**